patch/llpcPatchInOutImportExport.cpp

```diff
@@ -1904,10 +1896,9 @@ Value* PatchInOutImportExport::PatchFsGenericInputImport(
         pJ = ExtractElementInst::Create(pIJ, ConstantInt::get(m_pContext->Int32Ty(), 1), "", pInsertPos);
     }
 
-    std::vector<Value*> args;
-
-    std::vector<Attribute::AttrKind> attribs;
-    attribs.push_back(Attribute::ReadNone);
+    Attribute::AttrKind attribs[] = {
+        Attribute::ReadNone
+    };
```

Comment:
This can presumably be simplified further?

Reply:
Likewise.